src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -99,24 +101,25 @@ public ArmSubsystem() {
   public void setPosition(double angleDegrees) {
     targetDegrees = angleDegrees;
     armMotor.setControl(
-        new MotionMagicVoltage(Constants.Arm.ANGLE_TO_ENCODER_ROTATIONS(angleDegrees)));
+        controlRequest.withPosition(Constants.Arm.ANGLE_TO_ENCODER_ROTATIONS(angleDegrees)));
```

Comment:
Instead of creating a brand new `MotionMagicVoltage` request every time the setpoint changes, we can reuse a single instance and just update the position parameter.

This helps a lot with GC pressure, since the memory usage has effectively gone from _O(n)_ (where _n_ is number of robot loops) to _O(1)_.